### ISSUE TITLE
Have a test that verifies that we aren't leaving files in likely cache places (tmp, .cache, $HOME, etc)

### ISSUE THREAD

**Issue by yuvipanda** [jupyterhub/repo2docker#1121]
Should we have a test that verifies we aren't leaving files in likely cache places (tmp, .cache, apt places)?

_Originally posted by @minrk in https://github.com/jupyterhub/repo2docker/issues/1119#issuecomment-1021345645_

We cleaned up /tmp/downloaded_packages and got some space back in the image. This needs to be done at the exact same step the files are put there, otherwise it'll not be removed from the image.

**Comment by manics** [comment]
Probably enough to just check space used? E.g.
`[ $(du -sb /tmp/ | awk '{print $1}') -lt $MAX_SPACE_IN_BYTES ]`

**Comment by minrk** [comment]
> Probably enough to just check space used?

Yes! Once hooked up to the tests, I think we just need a list of directories to check: #1126
